resources/qgis_global_settings.ini

```diff
@@ -99,6 +99,12 @@ PostgreSQL\default_timeout=30
 # cause performance issues, as the records must all be loaded from the related table on display.
 maxEntriesRelationWidget=100
 
+# Path to default image to use for layout north arrows
+LayoutDesigner\defaultNorthArrow=:/images/north_arrows/layout_default_north_arrow.svg
```

Comment:
So can we can give a HTTP URL or a local filepath, without using a QRC file? Nice Thanks!

Reply:
Yes, fixed. Even base64 embedded north arrows are permitted :D